pxtblocks/fields/field_gridpicker.ts

```diff
@@ -348,7 +453,14 @@ export class FieldGridPicker extends Blockly.FieldDropdown implements FieldCusto
      * Create a dropdown menu under the text.
      * @private
      */
-    public showEditor_() {
+    public showEditor_(e?: MouseEvent) {
+        if (e) {
+            this.openingPointerCoords = {
```

Comment:
this should be disposed inside of `onClose_` right?

Reply:
After refactoring, this is now being cleared inside `disposeGrid`, which is called in the same places as `disposeTooltip`, including `onClose_`.